src/pages/profile/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { NextPage } from "next";
+import Page from "src/components/Page";
+import ProfileContent from "src/pageContent/Profile";
+
+const Profile: NextPage = () => {
+  return (
+    <Page bgImage="profile" title="Profile Page" subBanner description="">
```

Comment:
Also the linter, I think the linter would separate this into different lines, can check how to format on save

Reply:
Ok